app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -2115,6 +2116,7 @@ class BrowserTabFragment :
                     override fun process(featureName: String, method: String, id: String, data: JSONObject) {
                         when (method) {
                             "webShare" -> webShare(featureName, method, id, data)
+                            "permissionsQuery" -> permissionsQuery(featureName, method, id, data)
```

Comment:
You can directly call `viewModel.onPermissionsQuery(featureName, method, id, data)` here